content/en/blog/docker-multi-arch/index.md

```diff
@@ -0,0 +1,114 @@
+
+Otherwise, building for arm64 revealed no problems. The only needed change was that some of our server start scripts were missing [shebangs](https://en.wikipedia.org/wiki/Shebang_(Unix)) from the start of the script file. For yet an unknown reason, it prevented the shell from running the script properly on an arm64 container. The problem was fixed easily by adding the missing shebangs.
+
+The final step was to modify our GitHub Actions pipelines to build the images for the different architectures. Fortunately, Docker provides ready-made actions for setting up QEMU (*[setup-qemu-action](https://github.com/docker/setup-qemu-action)*) and buildx (*[setup-buildx-action](https://github.com/docker/setup-buildx-action)*), logging to the Docker registry (*[login-action](https://github.com/docker/login-action)*), and building and pushing the ready images to the registry (*[build-push-action](https://github.com/docker/build-push-action)*).
+
+We utilized the actions provided by Docker and [the release workflow](https://github.com/findy-network/findy-agent/blob/master/.github/workflows/release.yml) for findy-agent looks now like this:
+
+```yml
+name: release
+on:  
+  push:
+    tags:
+      - '*'
+jobs:
+
+  push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - run: echo "version=$(cat ./VERSION)" >> $GITHUB_ENV
+
+      - uses: docker/build-push-action@v2
+        with:
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: |
+            ghcr.io/${{ github.repository_owner }}/findy-agent:${{ env.version }}
+            ghcr.io/${{ github.repository_owner }}/findy-agent:latest
+          cache-from: type=registry,ref=ghcr.io/${{ github.repository_owner }}/findy-agent:latest
+          cache-to: type=inline
+          file: ./scripts/deploy/Dockerfile
+```
+
+The result was as expected, the actions take care of the building successfully. Building of the images is considerably slower with QEMU but luckily the build caches speed up the process a bit.
+
+Now have the needed variants for our service images in the registry. Furthermore, our colleague with the M1-Mac can run the agency successfully with his desktop.
```

Comment:
just for example of my first comment for the review, our colleague -> my colleague?